__tests__/app.test.js

```diff
@@ -133,6 +133,14 @@ describe('GET /api/articles/:article_id/comments', () => {
             })
         })
     })
+    test('GET: 200 sends an empty array when a given valid id with no comments associated with it', () => {
+        return request(app)
+          .get('/api/articles/2/comments')
+          .expect(200)
+          .then((response) => {
+            expect(response.body.msg).toBe('No comments found for this article.');
+          });
```

Comment:
I think for this - I would actually probably expect an empty array rather than a message :)